### ISSUE TITLE
Deleting a user results in weird behaviour

### ISSUE THREAD

**Issue by IIIuminator** [BjoernPetersen/MusicBot-desktop#49]
The delete request semms to work as the user isn't shown in the list anymore but the old token can still be used for login with all of its previous permissions intact. Trying to delete the user again results in AuthExpectations requesting Basic auth (which was used to send that request).

**Comment by BjoernPetersen** [comment]
That's actually not weird at all, it's just how JWT tokens work by default. They are valid until they expire and can not easily be revoked.

I will consider maintaining a "blacklist" in which issued tokens are stored and can be flagged as invalid.

**Comment by BjoernPetersen** [comment]
Another, perhaps complementary option would be to reduce the TTL of issued tokens, which is currently 3 hours iirc

**Comment by IIIuminator** [comment]
true, I'd be in favor of the blacklist approach as the current one has the issue that revoking permissions only goes into effect once the user's current JWT expired. I did a test and was able to delete queue entries even though I'd revoked my users permission to do so
